backend/Cargo.toml

```diff
@@ -20,7 +20,7 @@ strum = { version = "0.24.1", features = ["derive"] }
 log = "0.4.17"
 serde_json = "1.0"
 thiserror = "1.0.43"
-starky = { git = "https://github.com/0xEigenLabs/eigen-zkvm.git", rev = "a54965f" }
+starky = { git = "https://github.com/0xEigenLabs/eigen-zkvm.git", rev = "83c6c44" }
```

Comment:
With this we actually get rid of `ff_ce 0.11` and only use `ff_ce 0.12` everywhere.